matter_server/server/device_controller.py

```diff
@@ -71,18 +69,21 @@ async def initialize(self) -> None:
     async def start(self) -> None:
         """Handle logic on controller start."""
         # load nodes from persistent storage
-        nodes_data = self.server.storage.get(DATA_KEY_NODES, {})
-        for node_id_str, node_dict in nodes_data.items():
+        nodes: dict[str, dict] = self.server.storage.get(DATA_KEY_NODES, {})
+        for node_id_str, node_dict in nodes.items():
             node_id = int(node_id_str)
-            node = dataclass_from_dict(MatterNode, node_dict)
-            self._nodes[node_id] = node
-            # make sure to start node subscriptions
             try:
-                await self.subscribe_node(node_id)
-            except NodeNotResolving:
-                LOGGER.warning("Node %s is not resolving, skipping...", node_id)
-        # create task to check for nodes that need any re(interviews)
-        self._schedule_interviews()
+                node = dataclass_from_dict(MatterNodeData, node_dict)
+            except (TypeError, KeyError, AttributeError) as err:
+                # only accept this to happen if the schema changed
+                if node_dict.get("interview_version") == SCHEMA_VERSION:
+                    raise err
+                # set node to None will be automatically re-interview the node
+                node = None
+            self._nodes[node_id] = node
+        # setup subscriptions and (re)interviews as task in the background
+        # as we do not want it to block our startup
+        asyncio.create_task(self._check_subscriptions_and_interviews())
```

Comment:
We need to store a reference to tasks to make sure they are not garbage collected.

Reply:
Also for short running tasks ? As this one reschedules itself.

Reply:
yep

Reply:
ah good to know, I'll adjust. For my knowledge: was that changed in python3.11 ?

Reply:
adjusted